integration/service/spatialindexer/doc.go

```diff
@@ -0,0 +1,20 @@
+// package spatialindexer answers query of nearest points(place, point of interest) for conditions
```

Comment:
`package`->`Package` in comments, start with capitalized letter. 

See more details in https://github.com/golang/go/wiki/CodeReviewComments#package-comments

Reply:
Copy that.